packages/node/src/index.ts

```diff
@@ -333,6 +334,13 @@ async function compile(
         stream: preparedFiles[path].toStream(),
       });
 
+      if (!meta.compiledToCommonJS) {
+        meta.compiledToCommonJS = true;
+        console.log(
```

Comment:
Do we want this to be rendered to stdout or stderr? If stderr, then consider using `console.warn()` instead.